web/src/stream_list.ts

```diff
@@ -837,13 +839,46 @@ export function set_event_handlers({
 }: {
     on_stream_click: (stream_id: number, trigger: string) => void;
 }): void {
-    $("#stream_filters").on("click", "li .subscription_block", (e) => {
-        if (e.metaKey || e.ctrlKey || e.shiftKey) {
```

Comment:
The previous logic checked `shiftKey` too; why the change?

Reply:
there was a typo from my side.